webbpsf/tests/test_wfirst.py

```diff
@@ -108,3 +108,17 @@ def test_WFI_limits_interpolation_range():
     assert 'wavelength outside the range' in str(excinfo.value), (
         "FieldDependentAberration did not error on out-of-bounds wavelength"
     )
+
+def test_CGI_psf():
+    """
+    Just test that instantiating CGI works and can compute a PSF without raising
+    any exceptions
+    """
+    char_spc = wfirst.CGI()
+    char_spc.mode = 'CHARSPC'
+
+    #print('Reading instrument data from {:s}'.format(charspc._WebbPSF_basepath)
+    #print('Filter list: {:}'.format(charspc.filter_list))
+
+    monopsf = char_spc.calcPSF(nlambda=1, display=True)
+    wfirst.poppy.display_PSF(monopsf)
```

Comment:
Let's make the display optional - add a "display=False" default parameter, and then an "if display:" before the `display_psf` call. I'm not sure if this is 100% necessary but I think it makes it more reliable to run the test suite on Travis if we can leave the display parts turned off in that case.